live-assignments/century-finder/answer/spec/century.test.js

```diff
@@ -0,0 +1,13 @@
+const CenturyFinder = require("../app/js/century.js");
+const centuryFinder = new CenturyFinder();
+
+test("should correctly determine corresponding century for date", function() {
+	expect(centuryFinder.findCenturyOf(new Date())).toEqual("21st");
+
+	// remember months in JavaScript are 0-11, not 1-12
+	expect(centuryFinder.findCenturyOf(new Date(1776, 6, 4))).toEqual("18th");
+	expect(centuryFinder.findCenturyOf(new Date(1215, 5, 15))).toEqual("13th");
+	expect(centuryFinder.findCenturyOf(new Date(1945, 4, 8))).toEqual("20th");
+	expect(centuryFinder.findCenturyOf(new Date(2000, 0, 1))).toEqual("21st");
```

Comment:
![21st-century](https://github.com/user-attachments/assets/4e08904c-df74-42fa-a4ab-58be3d37ffd5)

Probably important to define in the instructions that it doesn't matter if they believe the 21st (or other) century starts in 2000 or 20001, as long as the code does what the interviewer and interviewee agreed

Reply:
That's a good point, with there being no 0AD it follows that the 100th year is the last year of the previous century and not the 1st of a new one.  I think I'll just update it all to respect this logic, as it adds another interesting dimension to the exercise.